.travis.yml

```diff
@@ -10,6 +10,7 @@ env:
   - TRIGGER_REPO_BRANCH: "master"
 
 before_install:
+  - nvm install 4
   - gem install jekyll -v 3.1.6
   - if [[ `npm -v` != 3* ]]; then npm i -g npm@3; fi
   - npm install -g bower grunt-cli
```

Comment:
jekyll is a gem and the version used is controlled by the Gemfile.lock file present int he repo.  nvm is the node version manager, and is used to install node 4 (which we use to build patternfly and angular patternfly).  We subsequently update npm to be version 3.  Note: nvm != npm.